src/test/java/uk/gov/companieshouse/itemgroupworkflowapi/integration/ItemGroupControllerIntegrationTest.java

```diff
@@ -0,0 +1,183 @@
+package uk.gov.companieshouse.itemgroupworkflowapi.integration;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.result.MockMvcResultHandlers;
+import uk.gov.companieshouse.itemgroupworkflowapi.model.DeliveryDetails;
+import uk.gov.companieshouse.itemgroupworkflowapi.model.Item;
+import uk.gov.companieshouse.itemgroupworkflowapi.model.ItemCostProductType;
+import uk.gov.companieshouse.itemgroupworkflowapi.model.ItemCosts;
+import uk.gov.companieshouse.itemgroupworkflowapi.model.ItemDescriptionIdentifier;
+import uk.gov.companieshouse.itemgroupworkflowapi.model.ItemGroupData;
+import uk.gov.companieshouse.itemgroupworkflowapi.model.ItemKind;
+import uk.gov.companieshouse.itemgroupworkflowapi.model.Links;
+import uk.gov.companieshouse.itemgroupworkflowapi.repository.ItemGroupsRepository;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+/**
+ * Integration tests the {@link uk.gov.companieshouse.itemgroupworkflowapi.controller.ItemGroupController} class.
+ */
+@SpringBootTest
+@AutoConfigureMockMvc
+@ComponentScan("uk.gov.companieshouse.itemgroupworkflowapi")
+public class ItemGroupControllerIntegrationTest {
+
+    private static final String EXPECTED_ORDER_NUMBER = "123456";
+    private static final String VALID_DELIVERY_COMPANY_NAME = "Delivery Test Company";
+    private static final String VALID_ITEM_COMPANY_NAME = "Item Test Company";
+    private static final String VALID_COMPANY_NUMBER = "IG-12345-67890";
+    public static final String REQUEST_ID_HEADER_NAME = "X-Request-ID";
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Autowired
+    private ObjectMapper mapper;
+
+    @Autowired
+    private ItemGroupsRepository repository;
+
+    @AfterEach
+    void tearDown() {
+        repository.deleteAll();
+    }
+
+    @Test
+    @DisplayName("Create successful itemGroup - 201 Created")
+    void createItemGroupSuccessful_201Created() throws Exception {
```

Comment:
This test is currently failing and the test method name should be updated to remove the underscore to be `createItemGroupSuccessful_201Created`